[2025-09-03T08:13:01.837+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-03T08:13:01.863+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: market_table_foreign.foreign_HOSE scheduled__2025-09-03T08:12:00+00:00 [queued]>
[2025-09-03T08:13:01.872+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: market_table_foreign.foreign_HOSE scheduled__2025-09-03T08:12:00+00:00 [queued]>
[2025-09-03T08:13:01.872+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 11
[2025-09-03T08:13:01.884+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): foreign_HOSE> on 2025-09-03 08:12:00+00:00
[2025-09-03T08:13:01.892+0000] {standard_task_runner.py:63} INFO - Started process 2710724 to run task
[2025-09-03T08:13:01.897+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'market_table_foreign', 'foreign_HOSE', 'scheduled__2025-09-03T08:12:00+00:00', '--job-id', '69439', '--raw', '--subdir', 'DAGS_FOLDER/dag_market_live.py', '--cfg-path', '/tmp/tmp_z61s2al']
[2025-09-03T08:13:01.902+0000] {standard_task_runner.py:91} INFO - Job 69439: Subtask foreign_HOSE
[2025-09-03T08:13:02.000+0000] {task_command.py:426} INFO - Running <TaskInstance: market_table_foreign.foreign_HOSE scheduled__2025-09-03T08:12:00+00:00 [running]> on host 1877907bc043
[2025-09-03T08:13:02.104+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='market_table_foreign' AIRFLOW_CTX_TASK_ID='foreign_HOSE' AIRFLOW_CTX_EXECUTION_DATE='2025-09-03T08:12:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-03T08:12:00+00:00'
[2025-09-03T08:13:02.105+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-03T08:13:02.119+0000] {foreign_HOSE.py:17} INFO - Kết nối DB
[2025-09-03T08:13:02.480+0000] {foreign_HOSE.py:42} INFO - Đã lưu foreign_HOSE
[2025-09-03T08:13:02.482+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-03T08:13:02.483+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-03T08:13:02.496+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=market_table_foreign, task_id=foreign_HOSE, run_id=scheduled__2025-09-03T08:12:00+00:00, execution_date=20250903T081200, start_date=20250903T081301, end_date=20250903T081302
[2025-09-03T08:13:02.550+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-03T08:13:02.582+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-03T08:13:02.585+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
